### ISSUE TITLE
Umbra: window flashes white at first open

### ISSUE THREAD

**Issue by adesurirey** [aCluelessDanny/typora-theme-ursine#63]
```
macOS: 10.15.6
Typora: Version 0.9.9.34.2 (4509)
Ursine: 2.0.1
```

Window flashes white for half a second when opening Typora with Ursine Umbra on. 

<details>
 <summary>GIF</summary>

![ursine-startup](https://user-images.githubusercontent.com/22000544/89982890-d837ab00-dc76-11ea-807a-957a4ac537fc.gif)
</details>

Doesn't seems to happen with Ursine Polar.

I also opened a similar issue on Typora as it affects multiple dark themes, but not all of them. You can find the list of dark themes I tested here: https://github.com/typora/typora-issues/issues/3776

**Comment by aCluelessDanny** [comment]
Duly noted and gonna check the themes you mentioned that doesn't have that flash for any ideas on how to circumvent that flash

**Comment by adesurirey** [comment]
Fixed in Typora Version 0.9.9.36.2 (4901)
